neurom/ezy/neuron.py

```diff
@@ -322,7 +323,8 @@ def _compare_neurites(self, other, neurite_type):
 
             return True if len(neurites1) == 0 and len(neurites2) == 0 else \
                    len(neurites1) - sum(1 for neu1, neu2 in
-                                        product(neurites1, neurites2) if neu1 == neu2) == 0
+                                        product(neurites1, neurites2)
+                                        if compare_trees(neu1, neu2)) == 0
```

Comment:
I mean in ezy neuron, whoever wants to know if trees are identical should be able to pass a binary predicate (such as the `compare_trees function`.)